### PR TITLE
Convert events page into light theme for providing a consistent user experience

### PR DESCRIPTION
**Pull request by y4izus**
For preview purposes before merging on master

### CODE REVIEW

File: pages/events/index.vue
Comment (delapuente) [inline pages/events/index.vue:200]: Don't do this, @techtolentino, but instead [modify the customized dark theme at `community-theme.scss`](https://github.com/Qiskit/qiskit.org/blob/master/assets/scss/carbon/community-theme.scss).

Let me provide more context: when we started, the last designer thought of giving a dark appearance to all the site so we started to [modify the Carbon theme](https://www.carbondesignsystem.com/guidelines/themes/overview/) for that end.

This is why you can find issues [regarding to how to apply two different themes to the same component](https://github.com/Qiskit/qiskit.org/blob/master/components/learn/TheLearningResourceList.vue#L194-L201).

Since we are moving to the light theme for all the pages, this kind of overriding is not longer needed. What we need to do is to use Carbon themes and customize to make it "white and purple".

Did I make my point?

**Review by delapuente — CHANGES_REQUESTED** [review]
I'm glad this is moving forward! Review my comments before merging.

**Comment by y4izus** [comment]
I think before merging we should fix [this](https://github.com/Qiskit/qiskit.org/issues/955)

**Review by delapuente — CHANGES_REQUESTED** [review]
@techtolentino I realize the theme SCSS file is not modified. We should try to implement light theme widgets by modifying the Carbon theme as suggested in https://github.com/Qiskit/qiskit.org/pull/951#discussion_r495666584. Are we planning to do this in a follow-up? If that's the case, please open the issue and link it here. I'd avice doing so and implementing during next sprint.

**Comment by techtolentino** [comment]
@delapuente - I created an issue here and we can prioritize it in the next sprint.

https://github.com/Qiskit/qiskit.org/issues/965

File: store/modules/events.ts
Comment (delapuente) [inline store/modules/events.ts:144]: I don't see this code being used. What is it for?

**Review by delapuente — CHANGES_REQUESTED** [review]
This is looking nice. A couple of things:
* I've open an issue regarding the arrows displayed in the dropdowns of the mobile version #1052.
* I'm seeing a lot of `div` tags. Remove all those `div` not really needed and remember to use `section`s to separate functional blocks. Try to use `div` for style/layout scaffolding and support only. 
* You should decide on #648 before continuing since here we are introducing a "wrapper" and that's a lot of different approaches already. :D

File: store/modules/events.ts
Comment (y4izus) [reply store/modules/events.ts:144]: We only have test for the getter function `filteredEvents`. We don't have yet test for the mutations functions and `updateFilterSet` is one of them. I think it could be better to create an issue to improve the code quality by adding tests for all the mutation function instead of creating now just one test for `updateFilterSet`. What do you think?

Comment (y4izus) [reply store/modules/events.ts:144]: Done [here](https://github.com/Qiskit/qiskit.org/pull/1054)

**Review by techtolentino — APPROVED** [review]
Thank you for all this, @y4izus - your updates LGTM.

**Review by korgan00 — APPROVED** [review]

**Pull request merged**
